packages/cli/src/commands/stats.ts

```diff
@@ -108,5 +121,4 @@ export async function handleStats(argv: StatsOptions, config: Config) {
   });
 
   printStats(statsAccumulator, path, argv.format);
-  printExecutionTime('stats', startedAt, path);
```

Comment:
I kept the opening debug line for the "stylish" output, but I'm not sure the execution time is that useful to anyone in any format so I removed it.

Reply:
I use the execution time in the default format regularly.

Reply:
That's a good data point. Only a couple of commands have it, and I didn't see how it was useful in the stats context. We can hold this change while I reinstate the feature and move it to the stylish output if you think many users will find it valuable.

Reply:
> I didn't see how it was useful in the stats context. 

The `stats` command has limited output yet parses and summarizes the entire API. The timing is useful because output is fixed and not a consideration when making comparisons (whereas `lint` timing changes based on the size of the problems and how many you display).

Reply:
Added it back in